README.md

```diff
@@ -158,6 +159,7 @@ Fill in the missing pieces in [this](https://github.com/Azure/kubernetes-keyvaul
             resourcegroup: "testresourcegroup"        # [REQUIRED for version < v0.0.14] the resource group of the KeyVault
             subscriptionid: "testsub"                 # [REQUIRED for version < v0.0.14] the subscription ID of the KeyVault
             tenantid: "testtenant"                    # [REQUIRED] the tenant ID of the KeyVault
+            nmiport: "nmiportnumber"                  # [OPTIONAL] port number of the NMI daemonset, will default to "2579"
```

Comment:
can we add `[OPTIONAL new in version >= v0.0.17]` to ensure users don't try it with the previous versions

Reply:
Ofcourse. It has been added to the properties table and the example.